skeleton/aws/variables.tf

```
@@ -23,3 +23,13 @@ variable "owner" {
   description = "The owner of the infrastructure, used to tag the resources, e.g. `acme-web`"
   type        = string
 }
+
+variable "app_name" {
+  description = "Application name"
+  type        = string
+}
```

we're using a variable names `var.namespace` in `advanced.ts`, please either rename this variable to `namespace` or replace the usuage of `namespace` in `advanced.ts` -> `app_name`.
```suggestion
variable "namespace" {
  description = "The namespace for the application infrastructure on AWS, e.g. acme-web"
  type        = string
}
```